### PR TITLE
Adding UI

### PR DESCRIPTION
**Pull request by czheng181920**
Adding UI

I left all of the functions alone (including connection manager)

Since you are going to change the join, create, etc button functions I didn't wire up any functions yet

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -24,7 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Monkeyracer</title>
```
Comment (charlin2) [inline client/public/index.html]: MonkeyRacer

File: client/src/App.jsx [hunk 2]
```diff
@@ -3,7 +3,11 @@ import './App.css';
...
 import { socket } from './socket'
-
+import Header from './components/Header';
+import Login from './components/Login';
+import Main from './components/Main';
+import Footer from './components/Footer';
+import { Routes, Route } from 'react-router-dom';
 
 function App() {
   const [passage, setPassage] = useState("");
```
Comment (charlin2) [inline client/src/App.jsx:14]: we can discuss which component this should live in

**Review by charlin2 — APPROVED** [review]
Include client commands in readme (npm start etc.)

**Comment by charlin2** [comment]
Can we vertically center the monkeyracer with the monkey logo in the header

File: client/src/App.jsx (hunk 2 above)
Comment (czheng181920) [reply client/src/App.jsx:14]: YUH

File: client/public/index.html (hunk 1 above)
Comment (czheng181920) [reply client/public/index.html]: ok done

**Pull request merged**